horace_core/sqw/PixelData/@PixelTmpFileHandler/PixelTmpFileHandler.m

```diff
@@ -0,0 +1,112 @@
+classdef PixelTmpFileHandler
+
+properties (Constant, Access=private)
+    DIRTY_PIX_DIR_BASE_NAME_ = 'sqw_pix%05d';
+    DIRTY_PIX_FILE_BASE_NAME_ = '%09d.tmp';
+    FILE_DATA_FORMAT_ = 'float32';
+end
+
+properties (Access=private)
+    dirty_pix_dir_ = '';
+    pix_id_ = -1;
+end
+
+methods
+
+    function obj = PixelTmpFileHandler(pix_id)
+        % Construct a PixelTmpFileHandler object
+        %
+        % Input
+        % -----
+        % pix_id   The ID of the PixelData instance this object is linked to.
+        %          This sets the tmp directory name
+        %
+        obj.pix_id_ = pix_id;
+        dirty_pix_dir_name = sprintf(obj.DIRTY_PIX_DIR_BASE_NAME_, obj.pix_id_);
+        obj.dirty_pix_dir_ = fullfile(tempdir(), dirty_pix_dir_name);
+    end
+
+    function raw_pix = load_page(obj, page_number, ncols)
+        % Load a page of data from the tmp file with the given page number
+        %
+        % Input
+        % -----
+        % page_number   The number of the page to read data from
```

Comment:
missing `ncols` argument description. Should it error or default to 1 if omitted?